FogCreek/Glitch-Community
# Add react axe to community

## Links
* https://add-react-axe-to-community.glitch.me/
* https://github.com/FogCreek/Glitch-Community-Work/issues/332

## How To Test:
* Open the dev tools of the remix and navigate around glitch. There are some 'minor' errors on the homepage that you'll see when you're logged in, might be others.
